Documentation/Hardware.md

```diff
@@ -1,8 +1,62 @@
 ## Notes on the various supported hardware
 
+
+### TS100
+
+TS100\* is a neat soldering iron:
+
+- can run from 9-25V DC;
+- provides a power range that is determined by the input voltage;
+- voltages below 12V don't overly work well for any substantial mass;
+- the default firmware can be found [here](https://www.minidso.com/forum.php?mod=viewthread&tid=892&extra=page%3D1).
+
+![](https://brushlesswhoop.com/images/ts100-og.jpg)
+
+
+### TS80
+
+TS80\* is a successor to TS100:
+
+- uses _Quick Charge 3.0_ / _QC3_ capable charger only (18W max);
+- doesn't support PD as it is not designed on the hardware level;
+- the default firmware can be found [here](https://www.minidso.com/forum.php?mod=viewthread&tid=3208&extra=page%3D1).
+
+![](https://core-electronics.com.au/media/catalog/product/4/2/4244-01.jpg)
+
+
+### TS80P
+
+TS80P\* is a successor to TS80:
+
+- supports _Quick Charge 3.0_ (_QC3_: 9V/3A, 18W max);
+- supports _Power Delivery_ (_PD_: 9V/3A & 12V/3A, 30W max)\*\*;
+- the default firmware can be found [here](https://www.minidso.com/forum.php?mod=viewthread&tid=4085&extra=page%3D1).
+
+\*\*: use valid PD device that supports 12V/3A as power source to get full 30W potential, otherwise the iron will fall back to QC/18W power mode.
```

Comment:
```suggestion
\*\*: use valid PD device that supports 12V/3A as power source to get full 30W potential, otherwise the iron will fall back to 9V/18W power mode.
```

Reply:
Got it, will fix that.